### ISSUE TITLE
hide() does not work when inline

### ISSUE THREAD

**Issue by okydk** [Simonwep/pickr#136]
Hey Simon,

When using `inline: true` the hide/show functionality doesn't work.
I'm trying to have it inline but hidden by default. :)

https://jsfiddle.net/f39vepm8/8/

```
Version (see Pickr.version): 1.2.7
Used bundle (es5 or normal one): both
Used theme (default is classic): all
```

**Comment by Simonwep** [comment]
Actually this is by-design but okay. Can be a feature 😄

**Comment by okydk** [comment]
Okay. :) It's just that the documentation under inline says:
> It's possible to hide it via .hide() anyway.
Would be grate if it didn't auto toggled `showAlways`.

**Comment by Simonwep** [comment]
oh, my bad - i'll fix that

**Comment by Simonwep** [comment]
Fixed in 854a33a39dea95b7eab945fe15d007b04a3baeac
